fem/fe_coll.hpp

```diff
@@ -1305,6 +1305,45 @@ class Local_FECollection : public FiniteElementCollection
    virtual int GetContType() const { return DISCONTINUOUS; }
 };
 
-}
+/// Radial basis function collection
+class KernelFECollection : public FiniteElementCollection
+{
+private:
+   int maxDim;
+   char d_name[32];
+   ScalarFiniteElement *Tr_Elements[Geometry::NumGeom];
+   ScalarFiniteElement *L2_Elements[Geometry::NumGeom];
+   int *SegDofOrd[2]; // for rotating segment dofs in 1D
+   int *OtherDofOrd;
+   bool ValidGeomType(Geometry::Type GeomType) const;
+
+public:
+   KernelFECollection(const int D,
+                      const int numPointsD,
+                      const int rbfType,
+                      const int order,
+                      const double h,
+                      const double faceFactor = 1.0, // between 0 (L2) and 1 (H1)
+                      const int intOrder = 2, // num integration points per 1d point
+                      const int distNorm = 2,
+                      const int mapType = FiniteElement::VALUE);
```

Comment:
This is the outward-facing interface to the RBF and RK finite elements. Where would be best to document these parameters?